march_rqt_robot_monitor/launch/march_rqt_robot_monitor.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
+    <node name="march_rqt_robot_monitor_node"
+          pkg="march_rqt_robot_monitor"
+          type="march_rqt_robot_monitor_node"
+          output="screen"
+          required="true" >
```

Comment:
[`required`](https://wiki.ros.org/roslaunch/XML/node) means that when this node crashes every other node should be shut down. I don't think this is necessarily the case here. The monitor is not necessary for the robot to be able to run. You could add `respawn`, which will cause roslaunch to respawn the node if it crashes.
```suggestion
    <node name="march_rqt_robot_monitor_node"
          pkg="march_rqt_robot_monitor"
          type="march_rqt_robot_monitor_node"
          output="screen"
          respawn="true">
```

Reply:
nice addition